freq/lizard_connector.py

```diff
@@ -265,16 +266,71 @@ def coord_uuid_name(self):
         :return: a dictionary with coordinates, UUIDs and names
         """
         result = {}
+        print(self.results)
         for x in self.results:
+            print(x)
             if x['uuid'] not in self.uuids:
+                geom = x.get('geometry') or {}
                 result[x['uuid']] = {
-                        'coordinates': x['geometry']['coordinates'],
+                        'coordinates': geom.get(
+                            'coordinates', ['','']),
                         'name': x['name']
                 }
                 self.uuids.append(x['uuid'])
         return result
 
 
+class TaskAPI(Base):
+    data_type = 'tasks'
+
+    def poll(self, url=None):
+        print('TaskAPI', url)
+        if url is None:
+            return
+        self.fetch(url)
+
+    @property
+    def status(self):
+        try:
+            logger.debug('Task status: %s', self.json.get("task_status"))
+            return self.json.get("task_status")
+        except AttributeError:
+            logger.debug('Task status: NONE')
+            return "NONE"
```

Comment:
Bedoel je de string `"NONE"` i.p.v. `None`?

Iets netter om eerst een `self.json.get()` te doen en dan te kijken of de uitkomst een 'None' is en daarop te reageren. Dan is het duidelijker wat de "hoofdlijn" van de methode is.